R/utils-get_code_dependency.R

```diff
@@ -231,7 +231,7 @@ extract_occurrence <- function(calls_pd) {
     # then in calls_pd there is a `SYMBOL_FORMALS` entry for that object.
     function_id <- x[x$token == "FUNCTION", "parent"]
     if (length(function_id)) {
-      x$id %in% get_children(x, function_id)$id
+      x$id %in% get_children(x, function_id[1])$id
```

Comment:
Why only the first item?

Reply:
We omit bodies of the calls that are function definitions. So we only need to know where the function starts, and if there is multiple function definitions inside the root function definition we only need it's start, so that's why taking the first element. When this was developed we only assumed there's gonna be a regular function definition and we aimed to seek for the start of the function